lib/ads/client_side_ad.js

```diff
@@ -207,6 +207,13 @@ shaka.ads.ClientSideAd = class {
     }
   }
 
+  /**
+   * @param {number} videoVolume
+   */
+  mute(videoVolume) {
```

Comment:
Okay, after some examining I agree that this method needs to exist, so that we can keep track of what the previous video volume is without actually setting the volume on the ad element.
However, this name is bad. It's very similar to the `setMuted` method, and doesn't really explain how the two methods are different.
Instead I'd call this something like "setInitialMuted", and add a comment to the JSDoc that it's required for a muted ad to start when autoplaying.

Reply:
Done!